src/components/speaking-event/speaking-event.scss

```diff
@@ -0,0 +1,13 @@
+.c-speaking-event {
+  position: relative;
+}
+
+.c-speaking-event__link::after {
+  bottom: 0;
+  content: '';
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  z-index: 1;
+}
```

Comment:
Can we include some focus styles?

Reply:
Also, it looks like we're missing the hover styles that were present in the prototype?

Reply:
Oh, oops. You are correct, thank you!

Reply:
I've added focus and hover styles! Thanks again, @tylersticka! 😄